src/worker/controller/actions/GetStateKeysAction.js

```diff
@@ -18,7 +18,12 @@ class GetStateKeysAction {
   }
 
   execute(params) {
-    let onResponse = params.onResponse;
+    let onResponse;
+    try {   // We account for the fact that params may be undefined
+      onResponse = params.onResponse;
+    } catch (e) {
+      onResponse = () => {};
+    }
     if (!onResponse) {
```

Comment:
Instead of doing try-catch here you should just replace this with:
```
if(params && params.onResponse) { 
	onResponse = params.onResponse 
	} else { 
		onResponse = () => {};
	}
```

Reply:
Corrected in 7db1069